Update Welcome Wagon for Venueless (#2218)
writethedocs/www: docs/include/conf/virtual/welcome-wagon.rst

Hello!

We’re |WelcomeWagonNames| your Welcome Wagon, and we’re glad you’re attending Write the Docs!
We realize a virtual conference might be totally new to you.
This guide will help you navigate it,
even if you’ve been to a Write the Docs conference before.

Whether this is your first Write the Docs conference or your first virtual Write the Docs conference,
we'll be here to help you make the most of it.

> We’d love to virtually meet you: feel free to send us a direct message at the virtual conference

Reviewer: ```suggestion
We’d love to virtually meet you: feel free to ask anything in the #helpdesk channel at the virtual conference (not private) or send us a direct messsage
```

> We’d love to virtually meet you: feel free to send us a direct message at the virtual conference
> if we can help make your time at the conference easier.

Author: ```suggestion
We’d love to virtually meet you:
feel free to ask anything in the #helpdesk channel at the virtual conference (not private)
or send us a direct message
if we can help make your time at the conference easier.
```

Guides
~~~~~~

You can prepare for the conference by reading through this guide.
Don’t worry if you forget any of it –

> we'll be waiting at the Helpdesk booth for any questions you might have.

Reviewer: ```suggestion
we'll be waiting at the #helpdesk channel on Venueless for any questions you might have.
```

Helpdesk
~~~~~~~~

Have a look at the `FAQ section <#faqs>`__ we put together based on questions we got at previous conferences.

If you still have questions, reach out the Welcome Wagon staff at the Helpdesk anytime.

> Find **Helpdesk** under *Channels*.

Reviewer: ```suggestion
Find **#helpdesk** under *Channels*.
```

If you don't see it there, select **Browse all channels** and join the **Helpdesk**.
A member of the Welcome Wagon will be online during conference hours to help you.

Message organizers
~~~~~~~~~~~~~~~~~~

If you want to contact one of the conference organizers,

> select **+Open a direct message** and search for an organizer's name.

Reviewer: ```suggestion
select **+Open a direct message** and search for an organizer's name.
However, using the #helpdesk channel is preferred, as not all organizers are available all the time.
```

Author: ```suggestion
select **+Open a direct message** and search for an organizer's name.
It's best to use the #helpdesk channel as not all organizers are available all the time.
```

such as the Sponsor Expo, conference schedule, and plan for Writing Day and the Unconference.

Don’t be afraid to explore!
You won’t interrupt anything by looking around.

Conference talks and Q&A
------------------------

Watching talks live
~~~~~~~~~~~~~~~~~~~

- There’s a talk schedule at the top of the sidebar under **Conference Schedule**.
  It offers times in three timezones.
- When it’s time for a talk you want to see, under *Stages* select **Stage**.
- If we have problems with the online platform, we will set up a back-up stream.
  If turned on, we will post links to the back-up stream in the conference chat and on the `#wtd-conferences channel in Slack <https://writethedocs.slack.com/archives/C1AKFQATH>`__.
- After a talk, feel free to tweet about it with the hashtag #writethedocs.

Asking questions and getting answers
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Immediately following each talk will be a moderated Q&A session.

To ask a question, during either the talk or the Q&A session:

1. In the Stage, open the **Questions** tab.
2. Select **Ask a question**.
3. Enter your question and select **Submit your question**.

> If there is time, a moderator will ask the question to the speaker.

Reviewer: ```suggestion
If there is time, a moderator will ask the question to the speaker.
Note that the moderator might put their own twist on the questions or their order.
```

View all questions asked and vote for those you find most interesting in the **Questions** tab.
If your question isn’t asked live, chat with the speaker later and make a connection!

Watching talks after the conference
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Don't worry if you miss something in a talk!

> Stage talks and Q&A sessions are recorded.

Reviewer: ```suggestion
Stage talks, Q&A sessions, and lightning talks are recorded.
```

- Start the session with a prompt:
  Think of a question that you have about the topic of your unconference and ask it.
  The conversation typically works itself out after that.
  If there’s a lull, ask the group if anyone else has a question to ask.
- People can join the session with audio and video, but others may just be in the chat.
  Be sure to check the session chat regularly and include people there in the conversation.
- Be on the lookout for people who aren’t joining in and invite them to speak.
  Don’t insist if they make it clear they want to listen only.

Join an unconference session
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- Starting Monday morning, check the `Unconference sign-up <{{unconf.url}}>`__
  to see if there are any sessions you are interested in joining.
  New ones are added all the time, so check back periodically.
{% if unconf and unconf.url %}
- At the time the session starts,
  find the table under *Channels* that corresponds to the one in the `Unconference schedule <{{unconf.url}}>`__.
{% else %}
- At the time the session starts,
  ind the table under *Channels* that corresponds to the one in the Unconference schedule.
{% endif %}

> - The session leader will begin the session when the group has gathered.

Reviewer: ```suggestion
- The session leader will begin the session when the group has gathered.
- You do not need to sign up anywhere to join a session.
```

Lightning talks
---------------

- A lightning talk is a five-minute talk
  where you quickly share a concept or bit of info you find interesting.
  Lightning talks are a great way to practice public speaking,
  get people excited about your Unconference session,
  and test interest in a conference proposal idea.
- If you are interested in giving a lightning talk, be prepared!
  Read our `guide to lightning talks <https://www.writethedocs.org/conf/{{ shortcode }}/{{ year }}/lightning-talks/>`__.

> - Watch the conference chat and your browser window
>   for an announcement that Lightning Talk sign-ups have opened and for directions on how to sign up.

Reviewer: ```suggestion
- Lightning talks will open on Sunday at the start of Writing Day,
  close a while before the first session to allow us to select talks, and then reopen right after the Monday Lightning 
  Talk session.
  Watch the conference chat and your browser window
  for an announcement that Lightning Talk sign-ups have opened and for directions on how to sign up.
```

Author: ```suggestion
- Sign-ups for Lightning Talks will open on Sunday at the start of Writing Day,
  close a while before the first session to allow us to select talks,
  and then reopen right after the Monday Lightning Talk session.
- Watch the conference chat and your browser window
  for an announcement that Lightning Talk sign-ups have opened and for directions on how to sign up.
```
